make/images

```diff
@@ -7,4 +7,9 @@ GIT_ROOT=${GIT_ROOT:-$(git rev-parse --show-toplevel)}
 cd "$GIT_ROOT"
 . "${GIT_ROOT}/.envrc"
 
-fissile build images
+if [ -z "${DOCKER_APP_VERSION:-}" ]; then
+  echo "DOCKER_APP_VERSION is not set"
+  exit 1
+fi
+
+fissile build images --add-label "version.cap=$DOCKER_APP_VERSION"
```

Comment:
Wouldn't it make more sense to always build, but not add the label if it's unknown?
```bash
fissile build images ${DOCKER_APP_VERSION:+--add-label "version.cap=${DOCKER_APP_VERSION}"}
```

Reply:
It's a requirement that we always label our images with a version, so it's best to fail if there's something wrong with it.